src/renderer/components/app.tsx

```diff
@@ -44,8 +44,8 @@ export class App extends React.Component {
     const clusterId = getHostedClusterId();
     logger.info(`[APP]: Init dashboard, clusterId=${clusterId}, frameId=${frameId}`)
     await Terminal.preloadFonts()
-    await clusterIpc.activate.invokeFromRenderer(clusterId, frameId);
-    await getHostedCluster().whenReady; // cluster.refresh() is done at this point
+    await clusterIpc.setFrameId.invokeFromRenderer(clusterId, frameId);
```

Comment:
ClusterStatus component will activate the cluster so at this point we only need to sync frame id.